src/index.ts

```diff
@@ -109,14 +111,13 @@ export default class OpentracingExtension<TContext extends SpanContext>
     }
 
     const name = info.fieldName || "field";
-    const parentSpan = context.span ? context.span : this.requestSpan;
```

Comment:
This is actually unrelated but also likely an issue. I used a similar approach in my own extension but this does not work as expected. The child spans end up being nested underneath the parent but they actually execute sequentially so trace timings become distorted.


Query:
```{
{ 
 parent: {
    field1
    field2
  }
}
```

Traces:
```
|--------------parent---------------------|
    |--childField1--||--childField2--|
```

What actually happens
```
|--parent--||--childField1--||--childField2--|
```